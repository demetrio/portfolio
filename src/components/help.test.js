import React from 'react';
import Help from './help';
import { render } from '@testing-library/react';

describe('Help', () => {
  it('renders correctly', () => {
    const { container } = render(<Help />);
    expect(container).toMatchSnapshot();
    expect(container).toBeDefined();
  });
});
