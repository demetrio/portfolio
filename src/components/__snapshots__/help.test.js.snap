// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Help renders correctly 1`] = `
<div>
  <button
    class="css-8ptk1h"
  >
     ðŸ™‚ Hey! You clicked 0 times!!
  </button>
</div>
`;
